<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="buttonFrame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>101</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRevoke">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNoteMode">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>note mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecover">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdMin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSec">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>620</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Sudoku Game</string>
    </property>
    <addaction name="actionEasy"/>
    <addaction name="actionNormal"/>
    <addaction name="actionHard"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCheating">
    <property name="title">
     <string>Cheating</string>
    </property>
    <addaction name="actionSimpleCheck"/>
    <addaction name="actionCheck"/>
    <addaction name="actionHint"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Answer"/>
   </widget>
   <widget class="QMenu" name="menuSolver">
    <property name="title">
     <string>Solver</string>
    </property>
    <addaction name="actionNewSolver"/>
    <addaction name="separator"/>
    <addaction name="actionSolveByComputer"/>
    <addaction name="actionSolveByPeople"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuCheating"/>
   <addaction name="menuSolver"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEasy">
   <property name="text">
    <string>New Easy Game</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>New Normal Game</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>New Hard Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSimpleCheck">
   <property name="text">
    <string>Simple Check</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionHint">
   <property name="text">
    <string>Hint</string>
   </property>
  </action>
  <action name="actionShow_Answer">
   <property name="text">
    <string>Show Answer</string>
   </property>
  </action>
  <action name="actionNewSolver">
   <property name="text">
    <string>New Solver</string>
   </property>
  </action>
  <action name="actionSolveByComputer">
   <property name="text">
    <string>Solve By Computer</string>
   </property>
  </action>
  <action name="actionSolveByPeople">
   <property name="text">
    <string>Solve By People</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
